### PR TITLE
Seal Remaining Identity-Games Session Safety Gaps

### PR DESCRIPTION
**Pull request by Dj-Shortcut**
Direct sessionId lookup now applies the same expiration guard as the userId and activeExperience paths, so expired identity-game sessions are treated as missing across all supported retrieval paths. Added a regression test covering getIdentityGameSessionBySessionId(expired) => null.

## Summary by CodeRabbit

* **Bug Fixes**
  * Improved locale parameter handling from URLs—now properly trims and handles whitespace in locale query parameters.
  * Enhanced session expiration detection for more reliable session state management.

* **Tests**
  * Added test coverage for locale parameter edge cases.
  * Added test coverage for session expiration scenarios.

### CODE REVIEW

File: server/_core/entryIntent.ts [hunk 1]
```diff
@@ -128,7 +128,7 @@ export function parseGameEntryIntent(input: {
     campaignId: params.get("campaignId") ?? undefined,
     creativeId: params.get("creativeId") ?? undefined,
     entryVariant: params.get("entryVariant") ?? undefined,
-    localeHint: input.localeHint ?? params.get("locale") ?? undefined,
+    localeHint: params.get("locale") ?? input.localeHint ?? undefined,
```
[the remark this answers is not in the document]
Comment (Dj-Shortcut) [reply server/_core/entryIntent.ts]: Fixed in 29be1dd. parseGameEntryIntent now ignores empty/whitespace-only locale query values instead of letting them mask the fallback locale hint, and there is a regression test covering ?locale=%20%20 -> input.localeHint.
